### PR TITLE
fakedeathed lings can now be empty cloned to prevent ling checking with cloning

### PR DESCRIPTION
**Pull request by ynot01**
## Why is this good for the game?

ling checking is unanimously retarded and if you try it you will be gibbed

## Intent of your Pull Request

im trying to save players from being gibbed

#### Changelog

:cl:  
rscadd: Added cloning for fakedeath lings
/:cl:

### CODE REVIEW

File: code/game/machinery/computer/cloning.dm [hunk 1]
```diff
@@ -468,6 +468,9 @@
 	else if (href_list["clone"])
 		var/datum/data/record/C = find_record("id", href_list["clone"], records)
 		var/empty = href_list["empty"]
+		var/datum/mind/M = C.fields["mindref"]
```
Comment (Bibby0110) [inline code/game/machinery/computer/cloning.dm]:
```suggestion
		var/datum/mind/M = locate(C.fields["mindref"])
```

**Comment by FatalEYES** [comment]
Ling checking needs to die in a fire i m o

**Comment by Moja10** [comment]
It will still be possible to ling check by shoving a ling brain into an MMI, correct? Just wondering

**Comment by JamieD1** [comment]
@ynot01 Make it so MMI ling check is no longer possible.

**Comment by ynot01** [comment]
sorry you lot, i tried to make this work but it just doesn't in testing, out of ideas, if someone more proficient in coding wants to take a crack at it be my guest

**Pull request closed without merging**